components/OneXP/OneXPPageSection.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <section class="page-section">
+    <div class="one-xp-page-section">
+      <div class="one-xp-page-section__container cds--grid">
+        <div class="one-xp-page-section__copy-section">
+          <h2 class="one-xp-page-section__copy-section__title">
+            <slot name="title" />
+          </h2>
+          <div class="one-xp-page-section__copy-section__features">
+            <div
+              v-for="feature in features"
+              :key="feature.title"
+              class="one-xp-page-section__copy-section__features__feature"
+            >
+              <h3 v-if="feature.title" v-text="feature.title" />
+              <p
+                v-for="description in feature.descriptionParagraphs"
+                :key="description"
+                class="one-xp-page-section__copy-section__features__feature__description"
+                v-text="description"
+              />
+              <div
+                class="one-xp-page-section__copy-section__features__feature__ctas"
+              >
+                <UiCta
```

Comment:
does our UiCta component allow the flexibility to override the hover state color? so that when hovered, it's `purple-80`?

![Screenshot 2023-08-11 at 6 09 45 AM](https://github.com/Qiskit/qiskit.org/assets/6276074/e859fe71-c7bc-47ef-bfd0-d1eaae0fdb52)

Reply:
> As discussed offline, these small design details are been discussed with the design team. It is known that our codebase won't reflect exactly the Figma design due to some structural differences, but that's OK for now.

Reply:
As discussed offline, these small design details are been discussed with the design team. It is known that our codebase won't reflect exactly the Figma design due to some structural differences, but that's OK for now.

In this case, "purple-70" was decided offline for the purple portion of the CTA gradient. The hover effect in our gradient buttons uses a CSS transition that moves the gradient to the right, coloring it all with the color that uses the left portion, so that would be "purple-70" for the way our component is coded.